src/app/feed/feed.module.ts

```diff
@@ -44,6 +45,10 @@ import { InViewportDirective } from '../shared//in-viewport.directive';
 		ReactiveFormsModule,
 
 		/**
+		 * The ng module for Forms
```

Comment:
If you checkout the file, We use multi-lined comments so that more information can be added later.